crates/graphql/src/schema/objects/nft.rs

```diff
@@ -158,8 +161,20 @@ impl Nft {
         &self.description
     }
 
-    pub fn image(&self) -> &str {
-        &self.image
+    pub fn image(&self) -> String {
+        let assets_cdn = "https://assets.holaplex.com";
+        let asset = AssetIdentifier::new(&Url::parse(&self.image).unwrap());
```

Comment:
Should check if both fields are Some.  It is technically not impossible that a link is ambiguous, in which case I think returning the raw URL is not out of the question.